spec/browserstack_helper.rb

```diff
@@ -1,40 +1,40 @@
-# -*- coding: utf-8 -*-
```

Comment:
Sorry, we're at cross-purposes there...

I meant the whole helper, not the UTF-8 declaration. Mea culpa.